packages/eslint-config-mc-app/has-jsx-runtime.js

```diff
@@ -1,5 +1,5 @@
 function hasJsxRuntime() {
-  if (process.env.DISABLE_NEW_JSX_TRANSFORM === 'true') {
+  if (process.env.ENABLE_NEW_JSX_TRANSFORM !== 'true') {
```

Comment:
Otherwise I fear that we make this a breaking change. As either users have to opt-out or migrate their code to node have `import React from 'react'`. We might be able to swap it to `DISABLE_` (opt-out) in the future and then remove it entirely. At least that's my current preferred thinking towards introducing it.

Reply:
Perfect, this is what I had in mind as well.